engine/src/test/java/io/zeebe/engine/util/client/WorkflowInstanceClient.java

```diff
@@ -76,6 +81,41 @@ public long create() {
     }
   }
 
+  public static class WorkflowInstanceCreationWithResultClient {
+    private StreamProcessorRule environmentRule;
+    private final WorkflowInstanceCreationRecord workflowInstanceCreationRecord;
+
+    public WorkflowInstanceCreationWithResultClient(
+        StreamProcessorRule environmentRule, WorkflowInstanceCreationRecord record) {
+      this.environmentRule = environmentRule;
+      this.workflowInstanceCreationRecord = record;
+    }
+
+    public long create(int requestStreamId, long requestId) {
```

Comment:
Could you please introduce new methods, like `withRequestStreamId` and `withRequestId` to keep the api understandable. I read the test before these changes and I thought these values `create(1, 1)` are for partition and key or something similar.